source/ranges.tex

```diff
@@ -4091,9 +4092,37 @@
 \pnum
 The name \tcode{views::take} denotes a
 range adaptor object\iref{range.adaptor.object}.
-Given subexpressions \tcode{E} and \tcode{F}, the expression
-\tcode{views::take(E, F)} is expression-equivalent to
-\tcode{take_view\{E, F\}}.
+Let \tcode{E} and \tcode{F} be expressions,
+let \tcode{T} be \tcode{remove_cvref_t<decltype((E))>}, and
+let \tcode{D} be \tcode{range_difference_t<decltype((E))>}.
+If \tcode{decltype((F))} does not model
+\tcode{\libconcept{convertible_to}<D>},
+\tcode{views::take(E, F)} is ill-formed.
+Otherwise, the expression \tcode{views::take(E, F)}
+is expression-equivalent to:
+
+\begin{itemize}
+\item
+If \tcode{T} is a specialization
+of \tcode{ranges::empty_view}\iref{range.empty.view},
+then \tcode{((void) F, \placeholdernc{decay-copy}(E))};
+
+\item
+otherwise, if \tcode{T} models
```

Comment:
another "models"; see http://lists.isocpp.org/lib/2020/02/15452.php

(similarly for the other occurrences below)

This seems to make the entire paper unimplementable, and it thus should be rejected.